src/portal/src/app/vulnerability-page/vulnerability-page.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="row">
     <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
-        <h2 class="custom-h2 vul-title">{{'VULNERABILITY.SINGULAR' | translate }}</h2>
+        <h2 class="custom-h2 vul-title">{{'CONFIG.VULNERABILITY' | translate }}</h2>
```

Comment:
why change this word?

Reply:
These two codes represent a word, but only a case difference.

Reply:
vulnerability ==>Vulnerability